### PR TITLE
chore: :wrench: Update for score_set flag

### PR DESCRIPTION
**Pull request by snoopy82481**
Updated to include `score_set` flag introduced in latest version of recyclarr.
Combined trash_id's in line with how a score_set is handled.
Cleaned up Adjustable section to be in line with the originating guide.

### CODE REVIEW

File: radarr/templates/anime-radarr.yml [hunk 1]
```diff
@@ -87,84 +88,38 @@ radarr:
...
-        quality_profiles:
-          - name: Remux-1080p - Anime
-            score: 300  # required as this differs from the standard scoring
-
-      - trash_ids:
           - af94e0fe497124d1f9ce732069ec8c3b  # WEB Tier 03
         quality_profiles:
           - name: Remux-1080p - Anime
-            score: 250  # required as this differs from the standard scoring
+          - name: Remux-1080p - Anime!
```
Comment (yammes08) [inline radarr/templates/anime-radarr.yml:107]: Don't think this is needed?

File: radarr/templates/anime-radarr.yml [hunk 2]
```diff
@@ -87,84 +88,38 @@ radarr:
...
           - 064af5f084a0a24458cc8ecd3220f93f  # Uncensored
         quality_profiles:
           - name: Remux-1080p - Anime
-            score: 0  # adjust as desired
-
-      - trash_ids:
-          - a5d148168c4506b55cf53984107c396e  # 10bit
-        quality_profiles:
-          - name: Remux-1080p - Anime
+            score: 101
```
Comment (yammes08) [inline radarr/templates/anime-radarr.yml:115]:
```suggestion
            score: 0  # adjust as desired
```

Comment (yammes08) [reply radarr/templates/anime-radarr.yml:115]: Default guide score is 0

File: radarr/templates/anime-radarr.yml [hunk 3]
```diff
@@ -87,84 +88,38 @@ radarr:
...
+          - name: Remux-1080p - Anime!
```
Comment (yammes08) [inline radarr/templates/anime-radarr.yml:116]: Don't think this is needed?

File: radarr/templates/anime-radarr.yml [hunk 4]
```diff
@@ -87,84 +88,38 @@ radarr:
...
             score: 0  # adjust as desired
 
       - trash_ids:
           - 4a3b087eea2ce012fcc1ce319259a3be  # Anime Dual Audio
         quality_profiles:
           - name: Remux-1080p - Anime
+            score: 10
```
Comment (yammes08) [inline radarr/templates/anime-radarr.yml:123]:
```suggestion
            score: 0  # adjust as desired
```

Comment (yammes08) [reply radarr/templates/anime-radarr.yml:123]: Default guide score is 0

File: radarr/templates/anime-radarr.yml [hunk 5]
```diff
@@ -87,84 +88,38 @@ radarr:
...
+          - name: Remux-1080p - Anime!
```
Comment (yammes08) [inline radarr/templates/anime-radarr.yml:124]: Don't think this is needed?

File: radarr/templates/anime-radarr.yml [hunk 6]
```diff
@@ -87,84 +88,38 @@ radarr:
...
             score: 0  # adjust as desired
```
Comment (yammes08) [inline radarr/templates/anime-radarr.yml:125]: Don't think this is needed (amended the score above)

**Review by yammes08 — CHANGES_REQUESTED** [review]
Made some comments/changes.

File: radarr/templates/anime-radarr.yml (hunk 1 above)
Comment (snoopy82481) [reply radarr/templates/anime-radarr.yml]: I fixed this in latest commit

File: radarr/templates/anime-radarr.yml (hunk 2 above)
Comment (snoopy82481) [reply radarr/templates/anime-radarr.yml]: I fixed this in latest commit

File: radarr/templates/anime-radarr.yml (hunk 3 above)
Comment (snoopy82481) [reply radarr/templates/anime-radarr.yml]: I fixed this in latest commit

File: radarr/templates/anime-radarr.yml (hunk 5 above)
Comment (snoopy82481) [reply radarr/templates/anime-radarr.yml]: I fixed this in latest commit

File: radarr/templates/anime-radarr.yml (hunk 6 above)
Comment (snoopy82481) [reply radarr/templates/anime-radarr.yml]: I fixed this in latest commit

File: radarr/templates/anime-radarr.yml (hunk 4 above)
Comment (snoopy82481) [reply radarr/templates/anime-radarr.yml]: The Anime Dual Audio and Uncensored are the only options that are user discretion.

**Review by yammes08 — APPROVED** [review]

**Comment by yammes08** [comment]
Really appreciate the PR - thanks!

**Pull request merged**